### ISSUE TITLE
BUG: Timetracks lost after start it

### ISSUE THREAD

**Issue by xstable** [owlysk/CodeTimer#7]
I had it now multiple times, but can't 100% reproduce it. 
From time to time I do the general way to start a timetrack... fill all the fields an click Enter (or with Mouse on Button `Start`). 

Then the screen switch back to the overview. 

Now there should be a red-button with the elapsed time... means the timetracker start to track the times. 
But in some cases, there is nothing at all. Only the Input-Field to start a new timetrack.

Happend multiple times now, so I filled this bug.

Is there a way to activate Developer-Console to see what's going wrong in this cases?

**Comment by xstable** [comment]
~~Not sure, but it might be the case, if you don't explicit choose the Customer and Project but use the pre-setting that was made if tab was open?!?~~
mhh no, it's not the issue. 
But today I have many trouble to start tasks. I currently have to do some tasks in a row (each task round about 5 - 10 Minutes).. and I found, that i nedd up to 5 times enter the new Timeentry to get it work once. 

I use the newest version of the app v1.0.9

**Comment by owlysk** [comment]
Hi,
I'm sorry about issues, you actually have. Can you please check, if task start in Kimai? 
If starts, try open some task in CodeTimer and go back to main page.  
If active task appears, that means CodeTimer need some delay to check active tasks.
If not starts in Kimai, maybe there is problem with description or connection to Kimai server.

Developer console can be opened with argument ```--window-enable-inspector```. 
If you are using Linux, go to app directory, open terminal and run command ```./CodeTimer-linux_x64 --window-enable-inspector```.

**Comment by xstable** [comment]
I've tried what you suggest and found that it also wasn't started at kimai.
If I try to start codeTimer with the inspector-parameter, application doesn't work anymore: 
![image](https://github.com/owlysk/CodeTimer/assets/1778068/e103ee41-5895-42a1-abf2-011e4f6dcd72)

On CLI I got this message too: 
> (process:33537): GLib-CRITICAL **: 09:58:50.647: g_uri_get_scheme: assertion 'uri != NULL' failed

But this also happend if I start it from console (cli) without the parameters. Only if I start it from KDE it seems to work.


-----

But here the error-logs from dev-console (as I've inject the parameters to the KDE-Starter):
![image](https://github.com/owlysk/CodeTimer/assets/1778068/e6dac574-e9e0-4b38-924e-4c30f641bbea)
